ui/app/js/controllers/rules_controller.js

```diff
@@ -321,6 +321,36 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
     });
   };
 
+  $scope.openNewScheduledDuplicateModal = function(rule) {
+
+    var modalInstance = $modal.open({
+      templateUrl: 'rule_scheduled_change_modal.html',
+      controller: 'NewRuleScheduledChangeCtrl',
+      size: 'lg',
+      backdrop: 'static',
+      resolve: {
+        scheduled_changes: function() {
+          return [];
+        },
+        sc: function() {
+          sc = angular.copy(rule);
+          sc["change_type"] = "insert";
+          return sc;
+        },
+        original_row: function() {
+          return rule;
```

Comment:
This needs a tweak too. `original_row` is meant to be "previous" version of the row, and is used to help style the Scheduled Change better. Because the created Scheduled Change is an insert, it doesn't have a previous version. This should return null, like openNewScheduledRuleModal does.